source/documentation/values/calculations.html.md.erb

```diff
@@ -161,42 +171,55 @@ CSS to unitless numbers:
-
-  Dart Sass 1.41.0 parses `min()` and `max()` functions as calculations, but
-  doesn't allow unitless numbers to be combined with numbers with units. This
-  was backwards-incompatible with the global `min()` and `max()` functions, so
-  that behavior was reverted.
-
-  [special functions]: ../syntax/special-functions
-<% end %>
+[Calculation Functions]: https://www.w3.org/TR/css-values-4/#math
+[`min()`]: ../modules/math#min
+[`max()`]: ../modules/math#max
+[`clamp()`]: ../modules/math#clamp
+[`round()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/round
+[`sin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sin
+[`cos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/cos
+[`tan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/tan
+[`asin()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/asin
+[`acos()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/acos
+[`atan()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan
+[`atan2()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/atan2
+[`pow()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/pow
+[`sqrt()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sqrt
+[`hypot()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/hypot
+[`log()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/log
+[`abs()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/abs
+[`exp()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/exp
+[`mod()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/mod
+[`rem()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/rem
+[`sign()`]: https://developer.mozilla.org/en-US/docs/Web/CSS/sign
+
+## [Special Cases]
 
 CSS added support for [`min()` and `max()` functions] in Values and Units Level
 4, from where they were quickly adopted by Safari [to support the iPhoneX]. But
-Sass supported its own [`min()`] and [`max()`] functions long before this, and
+Sass supported its own [`round()`], [`abs()`], [`min()`] and [`max()`] long before this, and
 it needed to be backwards-compatible with all those existing stylesheets. This
 led to the need for extra-special syntactic cleverness.
 
-[`min()` and `max()` functions]: https://drafts.csswg.org/css-values-4/#calc-notation
+[`min()` and `max()` functions]: https://www.w3.org/TR/css-values-4/#comp-func
 [to support the iPhoneX]: https://webkit.org/blog/7929/designing-websites-for-iphone-x/
+[`round()`]: ../modules/math#round
+[`abs()`]: ../modules/math#abs
 [`min()`]: ../modules/math#min
 [`max()`]: ../modules/math#max
 
-If a `min()` or `max()` function call is a valid calculation expression, it will
+If a calculation function call is a valid calculation expression, it will
 be parsed as a calculation. But as soon as any part of the call contains a
 SassScript feature that isn't supported in a calculation, like the [modulo
-operator], it's parsed as a call to Sass's core `min()` or `max()` function
+operator], it's parsed as a call to the appropriate Sass math
```

Comment:
```suggestion
operator], it's parsed as a call to the appropriate Sass math function
```

Also, please reflow paragraphs once you edit them. (Most editors have some built-in or plugin command for this.)